### DISCUSSION TITLE
An ongoing Serverpod tutorial series - Tech Startup from Scratch

### DISCUSSION THREAD

**Discussion by tylersavery** [serverpod/serverpod#2867] [Show and tell]
Hello Serverpod Friends!

So, I've been working on this large YouTube series (40 videos thus far) focusing on building a tech startup from scratch, from the first line of code to the release. I chose Serverpod as the framework to make it more approachable for my audience (generally frontend developers with little or no backend experience).

We are probably 70% through our MVP, so it is an excellent time to share it here!

Here's a link to the [playlist](https://www.youtube.com/watch?v=ja_ZuDKSpHU&list=PLgRx2Eap1Wm1qboogsOodOmwHqBvYbVSD)

With Serverpod Cloud coming soon, I'm hoping to get early access so I can familiarize myself with it and include it in the series.

I appreciate any feedback!

**Comment by HQHAN** [comment]
I am one of audience of the series and have been making SaaS following along. I hope Tyler can get an early access to promote the cloud feature of serverpod as early as possible !

**Comment by baudee** [comment]
This series is very helpful and helps a lot of people that want to use Serverpod. It offers a great visibility to the project. Would love to watch a video of Tyler using Serverpod Cloud !

**Comment by martinbakker** [comment]
Can really recommend this series. Good for starters or if you're experienced but come from another tech stack then Dart/Flutter. Really helps me to get up to speed and get a quick grasp on the do's and don't. Tyler is also very helpful on his discord channel when you get stuck.
Also kudos to Viktor and his team for creating "The Missing server for Flutter" (their words). Really enjoy the experience so far.

**Comment by vlidholt** [comment]
This is so awesome! 🤩 I gave you a shoutout in our social channels. We'll try to shortlist you for the Cloud beta!

**Reply by tylersavery** [reply]
Thanks for the s/o! Wasn't sure if you knew about this yet. I have posted it to Reddit a couple of times but based on what you guys are pumping out, I know you're busy!

**Comment by sWimmie** [comment]
I’m one of the many that follow this series and can not say enough about how helpful this has been. 
already have submitted my first app made with Serverpod to the AppStore and surely more will follow. 
Also notice that there is a subscription where many of us are. Helping out and sharing stories. But also Tyler is always there to help out on or off topic from the series. The community is great out there and would like to encourage you to join. 
But none of this would be possible without the amazing work from the Serverpod team what makes the development of complete apps a wonderful experience!

**Comment by super35link** [comment]
Hey Tyler and the Serverpod team!

I’ve been following your series and just wanted to say how much I’ve gotten out of it. The way you’ve approached Serverpod and building a startup from scratch has been incredibly valuable. The community and resources you've built around this have really helped fill in the gaps and made everything easier to follow.

I think getting early access to Serverpod Cloud would be an awesome opportunity, not just for you but for all of us following along. Seeing how you integrate it would be super helpful for everyone working on similar projects!

Can’t wait to see where things go from here!

**Comment by luedi69** [comment]
Hi Tyler,

Im on the second Video at about minute 15 and really love yoiur course so far.
Can't wait to find the time to watch the rest.

Greetings Thomas

**Comment by tananga** [comment]
🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳
